databases/week3/sql_week3.sql

```diff
@@ -0,0 +1,237 @@
+CREATE DATABASE meal_sharing;
+USE meal_sharing;
+
+CREATE TABLE meal(
+ id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+ title VARCHAR (255),
+ description TEXT,
+ location VARCHAR(255) NOT NULL,
+ event_day DATETIME NOT NULL,
+ max_reservations INT NOT NULL,
+ price DECIMAL,
+ created_date DATETIME DEFAULT NOW()
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE reservation(
+ id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+ number_of_guests INT NOT NULL,
+ created_date DATETIME DEFAULT NOW(),
+ contact_phonenumber INT NULL DEFAULT NULL,
+ contact_name varchar(255) NOT NULL,
+ contact_email varchar(255) NOT NULL,
+  meal_id INT UNSIGNED,
+CONSTRAINT fk_meal FOREIGN KEY (meal_id) REFERENCES meal(id) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
A more descriptive name for the foreign key constraint would be great. `fk_reservation_meal` for example

Reply:
It was so in my head, but missed it out while putting it down